staging/src/k8s.io/apiserver/pkg/server/filters/wrap.go

```diff
@@ -34,14 +34,12 @@ func WithPanicRecovery(handler http.Handler) http.Handler {
 }
 
 func withPanicRecovery(handler http.Handler, crashHandler func(http.ResponseWriter, *http.Request, interface{})) http.Handler {
+	handler = httplog.WithLogging(handler.ServeHTTP, httplog.DefaultStacktracePred)
```

Comment:
Avoid the `ServeHTTP` (applies to multiple locations)

```go
handler = httplog.WithLogging(handler, httplog.DefaultStacktracePred)
```

Reply:
Was necessary with the old type.